tests/infra/http/client/test_aiohttp_adapter.py

```
@@ -0,0 +1,91 @@
+import inspect
+from typing import Any
+from unittest.mock import AsyncMock
+
+from system_notification.infra.http.client.aiohttp_adapter import AioHttpAdapter
+
+
+class ClientSessionFake:
+    def __init__(self, status=200) -> None:
```

Ruff (0.13.3)

Unused method argument: `status`

(ARG002)